docs/UserGuide.md

```diff
@@ -177,6 +208,24 @@ Examples:
 - `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+### Sorting persons : `sort`
+
+Sorts the displayed list of persons by either name, student id or tutorial attendance.
+
+Format: `sort ORDER [n/][i/][tut/]`
+
+- `ORDER` indicates whether the sorted list is **ascending** or **descending**.
+- **Ascending** is represented with integer 1.
+- **Descending** is represented with integer -1.
+- `n/` indicates sorting according to name.
+- `i/` indicates sorting according to student id.
+- `tut/` indicates sorting according to tutorial attendance.
+
+Examples:
+
+- `sort -1 i/` sorts the person list in descending order according to student id.
```

Comment:
Format for tutorial here is wrong. Should include the tutorial number for sorting too.

Reply:
Oh my bad! Will amend immediately.